webapp/javascript/models/app.ts

```diff
@@ -0,0 +1,16 @@
+import { z, ZodError } from 'zod';
+import { Result } from '@webapp/util/fp';
+import { modelToResult } from './utils';
+
+export const appModel = z.object({
+  name: z.string(),
+});
+
+export const appsModel = z.array(appModel);
+
+export type Apps = z.infer<typeof appsModel>;
+export type App = z.infer<typeof appModel>;
+
+export function parse(a: unknown): Result<Apps, ZodError> {
+  return modelToResult(appsModel, a);
+}
```

Comment:
I don't think this is necessary anymore, see https://github.com/pyroscope-io/pyroscope/blob/main/webapp/javascript/services/tags.ts#L16 and https://github.com/pyroscope-io/pyroscope/blob/main/webapp/javascript/models/tags.ts for a reference